cmd/podman/shared/create.go

```diff
@@ -532,6 +532,10 @@ func ParseCreateOpts(ctx context.Context, c *GenericCLIResults, runtime *libpod.
 		annotations[splitAnnotation[0]] = splitAnnotation[1]
 	}
 
+	if c.IsSet("genseccomp") {
+		annotations["seccomp"] = "true"
```

Comment:
We really ought to set this to something very explicit. Most other annotation keys are in reversed-URL format with a very explicit name - something like `io.podman.seccomp-tracing` or similar. We should also make it a constant.